src/start-bot.ts

```diff
@@ -289,9 +286,6 @@ async function start(): Promise<void> {
         // TODO: Add new reactions here
     ];
 
-    // Triggers
-    let triggers: Trigger[] = [new OldPrefixTrigger()];
```

Comment:
Can the OldPrefixTrigger file be removed now? Along with the `oldPrefixUsed` embed?